recipes/DragAndDropHalogenHooks/web/index.css

```diff
@@ -1,4 +1,3 @@
 .otherCssNotInPurescript-Css {
-  background-color: lightblue;
   cursor: pointer;
```

Comment:
We could get rid of this last CSS workaround once https://github.com/purescript-contrib/purescript-css/pull/94 is merged.
Then we'll be able to host this recipe on TPS.

Reply:
Sounds good!